src/Resources/config/default_settings.yaml

```diff
@@ -1,4 +1,6 @@
 parameters:
+  ezplatform_graphql.schema.content.field_name.override:
+    id: id_
```

Comment:
Is `id` the only one field which might generate name collision?

Reply:
That's the only field that I'm aware of.

Reply:
OK, but please rememeber ot document this behaviour. // cc @ezsystems/documentation-team

Reply:
There are actually more, but they are all prefixed with `_`, making collisions less likely: https://github.com/ezsystems/ezplatform-graphql/blob/86f58702161da3060333459c55a677bf95d16d5f/src/Resources/config/graphql/DomainContent.types.yaml#L8-L31.